models/src/main/scala/coop/rchain/models/rholang/sorter/ScoreTree.scala

```diff
@@ -178,15 +178,16 @@ trait ScoreTree {
     final val BUNDLE_WRITE      = 307
     final val BUNDLE_READ_WRITE = 308
 
-    final val CONNECTIVE_NOT       = 400
-    final val CONNECTIVE_AND       = 401
-    final val CONNECTIVE_OR        = 402
-    final val CONNECTIVE_VARREF    = 403
-    final val CONNECTIVE_BOOL      = 404
-    final val CONNECTIVE_INT       = 405
-    final val CONNECTIVE_STRING    = 406
-    final val CONNECTIVE_URI       = 407
-    final val CONNECTIVE_BYTEARRAY = 408
+    final val CONNECTIVE_USED      = 400
+    final val CONNECTIVE_NOT       = 401
+    final val CONNECTIVE_AND       = 402
+    final val CONNECTIVE_OR        = 403
+    final val CONNECTIVE_VARREF    = 404
+    final val CONNECTIVE_BOOL      = 405
+    final val CONNECTIVE_INT       = 406
+    final val CONNECTIVE_STRING    = 407
+    final val CONNECTIVE_URI       = 408
+    final val CONNECTIVE_BYTEARRAY = 409
```

Comment:
This should not be needed. Also, we should be very cautious about changing the scores, as this may affect the order of execution / matching.